docs/guides/functions.ipynb

```diff
@@ -132,23 +133,47 @@
     "    ```python\n",
     "    from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
     "\n",
-    "    # After using the following code, go to your dashboard (https://quantum.ibm.com/)\n",
+    "    # After using the following code, go to your dashboard\n",
     "    # and expire your API token (click the refresh button in the API token field)\n",
```

Comment:
Do we know for sure that this woks in Legacy? In New, refreshing the API token actually just makes a new token.

Reply:
discussed offline